include/utility/json_utility.hpp

```diff
@@ -638,5 +638,59 @@ inline bool isBackupAndRestoreRequired(const nlohmann::json& i_sysCfgJsonObj)
     }
     return false;
 }
+
+/**
+ * @brief API to get PCA chip command with details taken from JSON.
+ *
+ * This API can be used to get PCA chip bind/unbind command for the given FRU
+ * path with the help of PCAChipAddress, busType and driverType tags taken from
+ * system config JSON.
+ *
+ * @param[in] i_fruPath - FRU path.
+ * @param[in] i_sysCfgJsonObj - System config JSON object.
+ * @param[in] i_action - Action parameter which takes "bind/unbind" as its
+ * value.
+ *
+ * @return On success returns PCA chip bind/unbind command. On failure returns
+ * empty string.
+ */
+inline std::string getPCAChipCmd(const std::string& i_fruPath,
+                                 const nlohmann::json& i_sysCfgJsonObj,
+                                 const std::string& i_action)
+{
+    if (i_fruPath.empty() || i_sysCfgJsonObj.empty() || i_action.empty())
+    {
+        return std::string();
+    }
+
+    try
+    {
+        if (i_sysCfgJsonObj["frus"].contains(i_fruPath))
+        {
+            const std::string& l_pcaChipAddr =
+                i_sysCfgJsonObj["frus"][i_fruPath].at(0).value("pcaChipAddress",
+                                                               "");
+            const std::string& l_busType =
+                i_sysCfgJsonObj["frus"][i_fruPath].at(0).value("busType", "");
+            const std::string& l_driverType =
+                i_sysCfgJsonObj["frus"][i_fruPath].at(0).value("driverType",
+                                                               "");
+
+            if (!l_pcaChipAddr.empty() && !l_busType.empty() &&
+                !l_driverType.empty())
+            {
+                std::string l_pcaChipCmd =
```

Comment:
Creating PCA chip command does not quality as a JSON utility. The information can be read from JSON but what to do with that information lies with the calling API.